provider/gce/gce_discover_test.go

```diff
@@ -1,12 +1,12 @@
 package gce_test
 
 import (
+	"github.com/hashicorp/go-discover"
 	"io/ioutil"
 	"log"
 	"os"
 	"testing"
 
-	discover "github.com/hashicorp/go-discover"
```

Comment:
IIRC `golangci-lint` likes to complain about undefined types for hyphenated aliases. Not sure whether it's complaining here but thought worth calling out.

Also super minor but I think formatting should place that dep w/ other `github.com` as before. NBD and not a blocker.

Reply:
reverted it, just in case 😅